File: pat-static/src/main/java/com/rectanglel/patstatic/predictions/PredictionsView.java
package com.rectanglel.patstatic.predictions;

import io.reactivex.SingleObserver;

/**
 * <p>Defines the View for Predictions</p>
 * <p>Created by epicstar on 10/14/16.</p>
 * @author Jeremy Jao
 * @since 78
 */

public interface PredictionsView<R> {
        SingleObserver<R> predictionsObserver();
}

File: pat-static/src/main/java/com/rectanglel/patstatic/predictions/PredictionsView.kt
package com.rectanglel.patstatic.predictions

import io.reactivex.SingleObserver

/**
 *
 * Defines the View for Predictions
 *
 * Created by epicstar on 10/14/16.
 * @author Jeremy Jao
 * @since 78
 */

interface PredictionsView<R> {
    fun predictionsObserver(): SingleObserver<R>
}
